Makefile

```diff
@@ -0,0 +1,9 @@
+run-slug:
+	npm run build && node src/slug.js
+
+test:
+	npm run test
+
+cow-slug:
+	pnpm run cow-slug
+
```

Comment:
The command for making the cow say a random phrase should be part of a .ts file, or, at most, present in Makefile so others could use it.

Reply:
Command included in Makefile 
say:
   pnpm run say

Reply:
![Captura de tela 2024-11-24 115905](https://github.com/user-attachments/assets/bc1f5276-c434-4e05-a7ae-9a0d6ce16d24)

Reply:
The command 'pnpm run say' outputs a cow saying "I´m a moooodule". The command 'pnpm run cow-slug' outputs a cow saying a random word. Task 8 asks to create a package script called 'cow-slug' that, when run, showa the cow saying a random slug. Therefore, I maintained the 'say' script created in Task 5 and created the 'cow-slug' script to output the cow saying a random word.